api/envoy/api/v2/core/base.proto

```diff
@@ -182,6 +184,12 @@ message Node {
   // See :ref:`the list of features <client_features>` that xDS client may
   // support.
   repeated string client_features = 10;
+
+  // Known listening ports on the node as a generic hint to the management server
+  // for filtering :ref:`listeners <config_listeners>` to be returned. For example,
+  // if there is a listener bound to port 80, the list can optionally contain the
+  // SocketAddress `(0.0.0.0,80)`. The field is optional and just a hint.
+  repeated Address listener_address_restriction_hint = 11;
```

Comment:
This PR looks good, just trying to see if we can maybe name this a bit more descriptively (I know I was responsible for the original suggestion :P)

How about `listener_bound_addresses`, `listening_addresses`, `listener_available_addresses`?

Reply:
I'll pick `listening_addresses` as the shortest of them all.